assets/css/sensei-course-theme/module-title.scss

```diff
@@ -0,0 +1,10 @@
+.sensei-course-theme-module-title {
+	font-family: 'Inter', sans-serif;
+	color: #000000;
+	font-size: 14px;
+	line-height: 17px;
```

Comment:
Could this `line-height` be relative too?

Reply:
Done. 👍